src/Library/Configs.php

```diff
@@ -103,6 +105,58 @@ public static function get( $key = null ) {
 		return $configs;
 	}
 
+	/**
+	 * Get the slug of the product that is loading the library.
+	 *
+	 * The configs include a "file" setting, which indicate which product is actually loading the
+	 * library. If the file is "boldgrid-backup/boldgrid-backup.php" for example, this method will
+	 * return "boldgrid-backup".
+	 *
+	 * @since 2.9.0
+	 *
+	 * @return string
+	 */
+	public static function getFileSlug() {
+		$file = explode( '/', self::get('file') );
+
+		return $file[0];
```

Comment:
Is that always going to have input to explode?  I rather use `return reset( $file );` to get the first index or FALSE.